### DISCUSSION TITLE
Adding EGLO/AwoX Connect-C Sensor

### DISCUSSION THREAD

**Discussion by AMDFreak2006** [fsaris/EspHome-AwoX-BLE-mesh-hub#39] [Q&A]
Currently the library only supports lights but no sensors.

From Eglo there is (since many years) the Connect-C sensor [click here](https://www.eglo.com/de/zubehoer-connect-sensor-97475.html) which combines a motion detection sensor with a brightness sensors (like most motion detection sensors...) and the Eglo/Awox BLE-interface. I have it in use since many years and it works quite flawlessly. However the possible things one can do with it in the original Awox app are sadly very limited.

Would it be possible to add it to the library?

Which information are needed for integration? I will of course try to provide everything which is needed.

**Comment by fsaris** [comment]
Did you check if the sensor is recognized by the component?

It should recognize it if the device is using the same mesh protocol. Will not be seen as a sensor but first step is to know if it's recognized at all.

How does this device show up in the AwoX app?
